core/server/common/src/main/java/alluxio/master/journal/JournalTool.java

```diff
@@ -15,9 +15,10 @@
 import alluxio.RuntimeConstants;
 import alluxio.conf.PropertyKey;
 import alluxio.conf.ServerConfiguration;
-import alluxio.master.CheckpointType;
 import alluxio.master.NoopMaster;
 import alluxio.master.journal.JournalReader.State;
+import alluxio.master.journal.checkpoint.CheckpointInputStream;
```

Comment:
Is this necessary?

Reply:
we moved the class from `journal` to `journal.checkpoint`